### PR TITLE
add kneed package

### PR DESCRIPTION
**Pull request by arvkevi**

### CODE REVIEW

**Comment by arvkevi** [comment]
Hi @conda-forge/staged-recipes does this look OK?

**Comment by arvkevi** [comment]
Hi @conda-forge/staged-recipes does this recipe look OK?

File: recipes/kneed/LICENSE [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Copyright (c) 2017, Kevin Arvai
```
Comment (beckermr) [inline recipes/kneed/LICENSE]: Please make a PR to the upstream repo to include the license file in the MANIFEST.in

File: recipes/kneed/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - setuptools
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+test:
+  imports:
+    - kneed
+  requires:
```
Comment (beckermr) [inline recipes/kneed/meta.yaml]: This section is not needed. The test env will install any dependencies required to run the package.

File: recipes/kneed/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - setuptools
```
Comment (beckermr) [inline recipes/kneed/meta.yaml]: You don't technically need to list setuptools here but it doesn't hurt.

File: recipes/kneed/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 39389c49b0cd9d976a3aaa8400aa9313bfb580818bc1ef60e0fcf4c9bb4f871e
+
+build:
+  noarch: python
```
Comment (beckermr) [inline recipes/kneed/meta.yaml]: Well done on the noarch package!

File: recipes/kneed/LICENSE (hunk 1 above)
Comment (arvkevi) [reply recipes/kneed/LICENSE]: IIUC, you mean in the `kneed` repo?

Comment (beckermr) [reply recipes/kneed/LICENSE]: yup!

**Comment by arvkevi** [comment]
Ok, made the requested changes, @conda-forge/staged-recipes does this look better?

**Pull request merged**